pkgs/by-name/os/oswald/package.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation {
+  pname = "oswald";
+  version = "4.103";
+
+  src = fetchFromGitHub {
+    owner = "googlefonts";
+    repo = "OswaldFont";
+    rev = "89795261ac9eeb9aa8cd99f43982c4e4b0e53261";
+    hash = "sha256-yoUduWHuuKDQaJnQ+CgeMw1vp2lgn/OVPokSDzEU7yk=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm444 fonts/ttf/*.ttf -t $out/share/fonts/truetype
+    install -Dm444 fonts/variable/*.ttf -t $out/share/fonts/variable
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "A reworking of the classic gothic typeface style";
```

ℹ️ According to section ["Meta attributes" in pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes)
`meta.description` should not start with a definite or indefinite article:
```suggestion
    description = "Reworking of the classic gothic typeface style";
```